chain/cmd/bandoracled2/handler.go

```diff
@@ -26,16 +26,25 @@ func handleTransaction(c *Context, l *Logger, tx tmtypes.TxResult) {
 
 	for _, log := range logs {
 		// TODO: Also handle IBC request packet here
-		messageType, err := GetEventValue(log, "message", "action")
+		messageType, err := GetEventValue(log, otypes.EventTypeMessage, otypes.AttributeKeyAction)
 		if err != nil {
 			l.Error(":cold_sweat: Failed to get message action type with error: %s", err.Error())
 			continue
 		}
-		if messageType != "request" {
+
+		if messageType == otypes.MessageTypeRequest {
+			go handleRequestLog(c, l, log)
+		} else if messageType == otypes.MessageTypeOraclePacket {
+			// Try to get request id from packet. If not then return error.
+			_, err := GetEventValue(log, otypes.EventTypeRequest, otypes.AttributeKeyID)
+			if err != nil {
```

Comment:
If err not nil, that means it's not an oracle packet, which is totally ok. I think we should debug log that we skip non-request packet.